projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,28 @@
+#Bartosz Habas
+
 FROM python:3.10
 
+WORKDIR /app
+
+COPY . /app
+RUN  pip install -r requirements.txt
+EXPOSE 8000 
+
+CMD ["python", "main.py"]
+
+
+
+#docker build --tag fiszki_backend .
+
+#docker run fiszki_backend 
+
+#docker run -p 8000 fiszki_backend 
```

Comment:
Prawie, niestety docker jednak wymaga podania portów we wzorze <port_dla_hosta>:<port_udostepniany_przez_kontener>